jkube-kit/generator/karaf/src/main/java/org/eclipse/jkube/generator/karaf/KarafGenerator.java

```diff
@@ -100,7 +100,8 @@ protected void addPortIfValid(List<String> list, String port) {
   }
 
 
-  private AssemblyConfiguration createDefaultAssembly() {
+  AssemblyConfiguration createDefaultAssembly() {
```

Comment:
Why is the method scope changed here? Is it just for testing?

Reply:
> Why is the method scope changed here? Is it just for testing?

to access it in test file

Reply:
Is it not possible to test this behavior by invoking `customize` method test?

Reply:
> Is it not possible to test this behavior by invoking `customize` method test?

not sure...should I change it?

Reply:
Could you please check this?

Reply:
This is still pending

Reply:
I have updated the PR using customize @manusa @rohanKanojia